src/main/java/com/wavefront/sdk/direct/ingestion/WavefrontDirectIngestionClient.java

```diff
@@ -101,6 +104,19 @@ public Builder flushIntervalSeconds(int flushIntervalSeconds) {
       return this;
     }
 
+    /**
+     * If configured, each batch is reported as one or more messages, with no message exceeding the
+     * specified number of kilobytes of data. By default, each batch is reported as a single
+     * message.
+     *
+     * @param kilobytes Maximum number of kilobytes of data that each reported message contains.
+     * @return {@code this}
+     */
+    public Builder messageSizeKilobytes(int kilobytes) {
```

Comment:
Just to keep the types the same, should we use `Integer kilobytes`? Also, `kiloBytes`?

Reply:
we can use `int` at both the places.

Reply:
Updated to use `int`.